packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/review/arm-networkanalytics.api.md

```diff
@@ -313,6 +318,10 @@ export interface ErrorAdditionalInfo {
     readonly type?: string;
 }
 
+// @public
+export interface ErrorAdditionalInfoInfo {
```

Comment:
new orphan model?

Reply:
It's an anonymous empty model https://github.com/qiaozha/typespec-azure/blob/a8cc75037f74fc221e3041c65cd56c0d1f4dd827/packages/typespec-azure-resource-manager/lib/common-types/types.tsp#L286 And because of point 6, we will generate the interface now.

Reply:
if so we should either link this to {} or just skip this generation considering no usage: https://github.com/Azure/autorest.typescript/blob/219814be49d41480c8d245e3402240c4c81ebbeb/packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/src/models/models.ts#L701-L706.

Reply:
Have you searched other places in the code? this is being used in the serializer and deserializer, and we do need to call the ser/deser even if anonymous model. I am keeping this comment for further discussion.